conf/fusesoc-configs/earlgrey.yml

```diff
@@ -9,11 +9,11 @@
 
 name: earlgrey
```

Comment:
I think we should split this file to at least 2 targets - ``synth`` and ``sim``, in similar to ibex:
- https://github.com/chipsalliance/sv-tests/blob/master/conf/fusesoc-configs/ibex-synth.yml
- https://github.com/chipsalliance/sv-tests/blob/master/conf/fusesoc-configs/ibex-sim.yml

In synthesis we should use ``lowrisc:systems:chip_earlgrey_nexysvideo`` target and current one: ``lowrisc:dv:chip_verilator_sim`` for simulation.

Reply:
Done